PINCache/PINDiskCache.m

```diff
@@ -744,9 +742,18 @@ - (id)objectForKeyedSubscript:(NSString *)key
                 PINDiskCacheError(error);
             }
             [self lock];
-          if (!self->_ttlCache) {
-            [self _locked_setFileModificationDate:now forURL:fileURL];
```

Comment:
Ok, actually perf appears to be worse in the set case because of the extra dispatchs clogging everything up, so I abstracted it out in those two cases.